spec/controllers/requests_controller_spec.rb

```diff
@@ -26,6 +26,12 @@
 # `rails-controller-testing` gem.
 
 RSpec.describe RequestsController, type: :controller do
+  # Authenticate an user
+  before do
+    user = FactoryBot.create :user
+    sign_in user
```

Comment:
If you want to save a line you could write `sign_in FactoryBot.create(:user)` as you don't actually use the user variable anywhere else.

Reply:
I implemented this your way. If some test need the user, it can always become a variable.